src/main/java/io/cryostat/recordings/RecordingArchiveHelper.java

```diff
@@ -121,8 +138,50 @@ public void deleteRecording(ConnectionDescriptor connectionDescriptor, String re
                 });
     }
 
+    public List<Map<String, String>> getRecordings() throws Exception {
```

Comment:
This probably shouldn't return a List of Maps, but instead a List of some more constrained type. The Maps used here always have the same set of keys, so this could/should instead be some class with fields named like those keys, and getters to access them.

Right now I could do `recordingArchiveHelper.getRecordings().get("nonsense key")` and this always returns `null`, but it should just be a compile-time error instead.